test/tests/services/PushRule.js

```diff
@@ -8,8 +8,8 @@ describe('PushRule', () => {
     });
     const pushRules = {
       upsert: true,
-      member_check: true
-    }
+      member_check: true,
+    };
```

Comment:
This is just a JSON literal does not need a semicolon. All you semicolon lovers 😅

Reply:
Well, normally your IDE should pick up `eslint` config and blame you for not following the standard.

Anyway, who cares, let's just make CI happy and green

Reply:
well my vscode did not pickup any eslint errors, did not have the extension installed 😢

Reply:
Maybe setup pre-commit hook to run "lint" script?